chungara-revista-de-antropologia-chilena.csl

```diff
@@ -1,205 +1,360 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" demote-non-dropping-particle="sort-only" sort-separator=", " et-al-use-first="2" version="1.0" default-locale="es-CL">
-  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" and="text" delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="." demote-non-dropping-particle="sort-only" default-locale="es">
```

Comment:
@e42mercury, can you clarify what locale this style will be used with? The "default-locale" setting must either be absent (for polyglot styles that will be used in multiple languages), or set to a specific locale (e.g. "es-ES" for Spanish or "es-CL" for Chilean Spanish).

Reply:
is there a general locale for Latin American Spanish? that would be best. Otherwise, es-CL is fine.

Reply:
We currently only have CSL locales for "es-ES" (Spanish) and "es-CL" (Spanish - Chile): https://github.com/citation-style-language/locales/. We can add additional Spanish locales, but we typically only do so if there are clear geographic differences in the language. Individual styles can also always override translations from the CSL locale files.